src/GVF/GVF.jl

```diff
@@ -26,28 +26,26 @@ get(gvf::AbstractGVF, state_t, action_t, state_tp1, preds_tilde) =
 get(gvf::AbstractGVF, state_t, action_t, state_tp1) =
     get(gvf::AbstractGVF, state_t, action_t, state_tp1, nothing, nothing)
 
+function cumulant(gvf::AbstractGVF) end
```

Comment:
Are these functions no-ops? Should we raise an error if we get into this state?

Reply:
We might want to throw an error. I'm not sure if this is considered a no-op, but results in a return of nothing.

Reply:
Gotcha. Yup, I'd vote in favor of throwing an error. Especially for consistency with the other abstract methods in the lib.